suite-native/graph/src/hooks.ts

```diff
@@ -48,17 +58,26 @@ const useWatchTimeframeChangeForAnalytics = (
                 });
             }
         }
-    }, [hoursToHistory, networkSymbol, isFirstRender]);
+    }, [timeframeHours, networkSymbol, isFirstRender]);
 };
 
 export const useGraphForSingleAccount = ({
     accountKey,
     fiatCurrency,
 }: CommonUseGraphParams & { accountKey: AccountKey }) => {
-    const account = useSelector((state: any) => selectAccountByKey(state, accountKey));
-    const [hoursToHistory, setHoursToHistory] = useState<HoursValue>(24);
+    const dispatch = useDispatch();
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+    const accountGraphTimeframe = useSelector((state: GraphSliceRootState) =>
+        selectAccountGraphTimeframe(state, accountKey),
+    );
+
+    const handleSelectAccountTimeframe = (timeframeHours: TimeframeHoursValue) =>
```

Comment:
useCallback

Reply:
Are the callbacks here really necessary? I've checked with the DebugView turned on, and that does not prevent any extra re-renders.

Maybe what better question is: What is your rule of thumb to use `useCallback` hook? Because I am trying to not use it extensively, but you commented already few times on my PRs that I should use it. I just probably need an explanation.

Reply:
It depends where and how this hook will be used, since this package is basically library that can be used anywhere someone in future may use this hook in different context or pass it to useEffect hook for example which will cause component rerender.

If you are writing some custom component for one-time use on some screen is OK to not use useCallback because you can easily verify in that one usage that it's not issue.

Reply:
Callbacks added in 5ac6deca7.